front/src/hooks/useLoadAndSetGlobalPreferences.ts

```diff
@@ -21,11 +21,8 @@ export const useLoadAndSetGlobalPreferences = () => {
 
   useEffect(() => {
     // run query only if the access token is in the request header from the apollo client and the base is not set
-    if (user && !globalPreferences.selectedBase?.id) {
-      runOrganisationAndBasesQuery();
-    }
-  }, [runOrganisationAndBasesQuery,
-    user, globalPreferences.selectedBase?.id]);
+    if (user && baseId === "0") runOrganisationAndBasesQuery();
```

Comment:
This would mean that the query is only run if the url does not start with `/bases/<base_id>` and , thus the baseId === "0", correct. However, we want the globalPreferences filled also for the cases that a baseId is passed in the url. Therefore, I would put this back where it was.